romancal/jump/tests/test_jump_step.py

```diff
@@ -110,6 +120,10 @@ def _setup(ngroups=10, readnoise=10, nrows=20, ncols=20,
 
 
 @pytest.mark.parametrize("max_cores", MAXIMUM_CORES)
+@pytest.mark.skipif(
```

Comment:
Since we're using this frequently, maybe we ought to create a fixture?

```python
@fixture(scope="module")
def require_crds():
  if os.environ.get("CI") == "true":
    pytest.skip("Roman CRDS servers are not currently available outside the internal network")
    
    
def test_one_CR(generate_wfi_reffiles, max_cores, setup_inputs, require_crds):
  # ...
```